Riot/Modules/Common/Recents/RecentsViewController.m

```diff
@@ -2253,6 +2258,17 @@ - (void)searchBar:(UISearchBar *)searchBar textDidChange:(NSString *)searchText
                               state:UIControlStateNormal];
 }
 
+- (void)searchBarCancelButtonClicked:(UISearchBar *)searchBar
+{
+    [self.recentsSearchBar resignFirstResponder];
+    [self hideSearchBar:YES];
+    self.recentsTableView.contentOffset = CGPointMake(0, self.recentsSearchBar.frame.size.height);
```

Comment:
I wonder if it would make more sense to leave the placeholder search bar visible rather than hiding it? That would be more familiar when you compare it to Mail/Messages etc.

Reply:
I think iOS doesn't have a consistent behaviour for this. For example at least for me it's auto visible for Mail but not Messages or Settings. Let's keep it as is, we can think about it more in ElementX with the design team.

Reply:
Both of those apps keep it visible for me when tapping the cancel button, but makes sense to wait until we have a design team again 😄